Sofa/Component/Constraint/Lagrangian/Correction/src/sofa/component/constraint/lagrangian/correction/PrecomputedConstraintCorrection.inl

```diff
@@ -221,19 +221,33 @@ void PrecomputedConstraintCorrection<DataTypes>::bwdInit()
 
     const SReal dt = this->getContext()->getDt();
 
-    invName = f_fileCompliance.getFullPath().empty() ? buildFileName() : f_fileCompliance.getFullPath();
+    invName = f_fileCompliance.getFullPath();
+    bool complianceLoaded = false;
+    if (!invName.empty())
+    {
+        complianceLoaded = loadCompliance(invName);
+        if (!complianceLoaded) {
+            msg_error() << "A fileCompliance was given at path: " << invName << ", but could not be loaded.";
+            invName = buildFileName();
```

Comment:
```suggestion
        if (!complianceLoaded) 
        {
            msg_error() << "A fileCompliance was given at path: " << invName << ", but could not be loaded.";
            invName = buildFileName();
```

Reply:
and I guess there should be a componentstate set to Invalid but that is not the point of the PR

Reply:
not necessary, because if file is not found the component will do the precomputation as if no file were given.
This is just to add a warning to to know why the scene loading is so slow sometimes ;)